src/_includes/about_asset.njk

```diff
@@ -0,0 +1,5 @@
+<section class="relative inline-block text-3xl md:text-4xl lg:text-5xl md:mr-10 lg:mr-64 pointer-events-none">
+    <div>
+  ASSET
+  </div>
+</section>
```

Comment:
It's good that we know how to do this for when we have to put an asset here, but we probably shouldn't merge dead/placeholder code into master. Can we remove this (unless @tshiUCSB has the  asset ready already)

Reply:
Will remove